enhancements/baremetal-IPI/baremetal.md

```diff
@@ -0,0 +1,220 @@
+---
+title: Neat-Enhancement-Idea
+authors:
+  - "@sadasu"
+reviewers:
+  - TBD
+  - "@abhinavdahiya"
+approvers:
+  - TBD
+  - "@abhinavdahiya"
+creation-date: 2019-11-06
+last-updated: 2019-11-06
+status: implemented
+---
+
+# Adding Baremetal IPI capabilities to OpenShift
+
+This enhancement serves to provide the context for a whole slew of features
+and enhancements that will follow to make Baremetal IPI deployments via
```

Comment:
Can we add a definition for `IPI` at the top here?

Reply:
Done.